jupyter_drives/tests/test_handlers.py

```diff
@@ -79,3 +79,39 @@ async def test_ListJupyterDrives_s3_missing_credentials(jp_fetch, set_s3_base):
         # Then
         error = json.loads(response.body)
         assert error == "No AWS credentials provided."
+
+@pytest.fixture
```

Comment:
FYI if you want to skip a test you should do

```suggestion
@pytest.mark.skip(reason="FIXME")
```

See the [documentation](https://docs.pytest.org/en/6.2.x/skipping.html#skipif)